src/main/java/core/basesyntax/HasArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface HasArea {
+    double area();
```

Comment:
According to the checklist, don't use verbs for class/interface names. Consider renaming the `HasArea` interface to something that follows the noun convention, such as `AreaCalculable`.